docs/specification.md

```diff
@@ -214,6 +214,7 @@ These roles apply to the whole contract (hence "global"):
   - Can drop any proposal at any time.
   - Cannot be an implicit address, in other words it must be a contract.
   - There always must be exactly one `guardian`.
+  - Can be updated via a proposal.
```

Comment:
Not sure if we should mention it here, since it's a feature of `registry` and `treasury` but not of `baseDAO` in general.

Edit: never mind, given the change of the decision lambda return type i guess it's actually worth mentioning it here (leaving this comment just for reference)